packages/frontend/src/app/services/search/search.service.spec.ts

```diff
@@ -76,53 +76,55 @@ describe('SearchService', () => {
 
   describe('search', () => {
     it('should return the fetched repositories', done => {
-      const search: Search = {query: '', filters: []};
+      service.search('repo', 'org', []).subscribe(reponse => {
+        expect(reponse.repos.length).toEqual(repositories.repos.length);
 
-      service.search(search, 'org').subscribe(repos => {
-        expect(repos.length).toEqual(repositories.repos.length);
-
-        repos.forEach((repo, index) => {
+        reponse.repos.forEach((repo, index) => {
           expect(repo.name).toEqual(repositories.repos[index].name);
         });
         done();
       });
     });
 
-    it("should add the 'order by priority' filter to the search if itsn't already present", done => {
-      const search: Search = {query: '', filters: []};
+    it("should add the 'order by priority' filter to the search if itsn't already present", fakeAsync(() => {
```

Comment:
do we have a test that makes sure the value of `order by priority` changes to a different `priority` value when the component is interacted with?

Reply:
Yes, in the [filters component](https://github.com/GoogleCloudPlatform/flaky-service/blob/master/packages/frontend/src/app/filters/filters.component.spec.ts#L63L76).